src/_constants/ActionTypes.js

```diff
@@ -40,3 +40,6 @@ export const SIGNIN_FAILED = 'SIGNIN_FAILED';
 
 export const SERVER_DATA_ACCOUNT_LIMITS = 'SERVER_DATA_ACCOUNT_LIMITS';
 export const SERVER_DATA_ACCOUNT_SETTINGS = 'SERVER_DATA_ACCOUNT_SETTINGS';
+
+export const NEXT_VIDEO = 'NEXT_VIDEO';
+export const PREVIOUS_VIDEO = 'PREVIOUS_VIDEO';
```

Comment:
Oh, we will ;) 
Just the way we notify the change internally will be different.

You will also need a <VideoList> component, custom created by you, just reuse the HTML I generate in the blog itself. Will be enough for now.